components/auth/BankIdAuth.tsx

```
@@ -521,29 +685,42 @@ export function BankIdAuth({ mode, onComplete, hero = false }: BankIdAuthProps)
+          )}
         </>
       )}
 
       {isMobile() && (
         <div className="flex flex-col items-center gap-2">
           <Smartphone className="h-8 w-8 text-muted-foreground" />
-          <p className="text-sm text-muted-foreground">Öppnar BankID-appen...</p>
+          {/* A resumed tab never launched anything, so saying "opening the
+              BankID app" would be a lie followed by silence. */}
+          <p className="text-sm text-muted-foreground">
+            {launchedRef.current
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/components/auth/BankIdAuth.tsx:712:14
  710 |               BankID app" would be a lie followed by silence. */}
  711 |           <p className="text-sm text-muted-foreground">
> 712 |             {launchedRef.current
      |              ^^^^^^^^^^^^^^^^^^^ Cannot access ref value during render
  713 |               ? 'Öppnar BankID-appen...'
  714 |               : t('bankid_finish_in_app')}
  715 |           </p>

(react-hooks/refs)